### ISSUE TITLE
Update licensing on the website

### ISSUE THREAD

**Issue by aboullaite** [DevC-Casa/stateofdev.ma#27]
We need to specify/separate the licensing for the code and for raw data.
On the github repo we mentioned MIT license, however on the website it still BY-NC-SA 2.0!
We should probably opt for [ODC-ODbL](https://opendatacommons.org/licenses/odbl/) for raw data

**Comment by AbderrahimSoubaiElidrissi** [comment]
Create a LICENSE file in root project and put the content of [MIT](https://opensource.org/licenses/MIT) Licence.
 Please make sure to change:
- <YEAR> by 2021
-  <COPYRIGHT HOLDER> by DevC Morocco 

@aboullaite @yjose Please confirm COPYRIGHT HOLDER

**Comment by aboullaite** [comment]
@MidouWebDev please yourself :)

**Comment by aboullaite** [comment]
Yes! There are 2 licenses here. One for the website and one for the data.
MIT is the license for the website. You need probably to add another one ODC-ODbL for raw data under survey. 
The license holder should be Geeksblabla I believe

**Comment by bilalix** [comment]
What we can do is:
* Keeping the MIT `LICENSE` file in the root level that outlines the terms of the chosen license for the repository as a whole. 
* Add another license file under `./results` stating that the data is under [ODC-ODbL](https://opendatacommons.org/licenses/odbl/) license.
* Additionally, in the README file, we can clarify that the code and data are separately licensed, and provide links to the individual licenses for each.

I think this approach covers both data and code sensibly.
I'll create a PR with the changes and we can take it from there.

**Comment by bilalix** [comment]
@aboullaite, please feel free to close this issue.
